docs/hub/keras.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Keras
+---
+
+# Using Keras at Hugging Face
+
+`Keras` is an open-source machine learning library that leverages PyTorch and Python to provide high-level components to train neural networks with state-of-the-art outputs on all modalities.
+
+## Exploring Keras in the Hub
+
+You can find `Keras` models by filtering at the left of the [models page](https://huggingface.co/models?library=keras&sort=downloads). All models on the Hub come up with useful features such as an automatically generated model card and metadata tags that help for discoverability. If provided by the model owner, TensorBoard logs are hosted on the Keras repositories as well.
+
+
+## Using existing models
+
+The `huggingface_hub` library is a lightweight Python client with utlity functions to download models from the Hub.
+
+```
+pip install huggingface_hub["tensorflow"]
+```
+
+Once you have the library installed, you just need to use the `from_pretrained_keras` method. 
+
+```py
+from huggingface_hub import from_pretrained_keras
+
+model = from_pretrained_keras("keras-io/mobile-vit-xxs")
+prediction = model.predict(image)
+prediction = tf.squeeze(tf.round(prediction))
+print(f'The image is a {classes[(np.argmax(prediction))]}!')
+
+# The image is a sunflower!
+```
+
+
+If you want to see how to load a specific model, you can click `Use in keras` and you will be given a working snippet that you can load it! 
+
+![snippet](docs/assets/hub/keras_snippet1.png)
+![snippet](docs/assets/hub/keras_snippet2.png)
+
+## Sharing your models
+
+You can share your `keras` models by using the `push_to_hub_keras` method. This will generate a model card that includes your model’s hyperparameters, plot of your model and couple of sections related to the usage purpose of your model, biases the training data might have and limitations about putting your model to production, puts the metrics of your model in a JSON if model has a history. 
```

Comment:
The second sentence in this paragraph is really long and ends up being a bit confusing. Is it possible to split it into two or three?

Reply:
sure!